lib/app.dart

```diff
@@ -0,0 +1,97 @@
+import 'dart:html' as html;
+
+html.DivElement get content => html.querySelector("#content");
+html.DivElement get container => html.querySelector("#container");
+html.DivElement get interactive => html.querySelector("#interactive");
+html.DivElement get progress => html.querySelector("#progress");
+html.ButtonElement get prevButton => html.querySelector("#prev-button");
+html.ButtonElement get nextButton => html.querySelector("#next-button");
+
+class App {
+  int currentPage = 0;
+  int maxPageCount = 10;
```

Comment:
If you're making all of the other methods private, you may want to make these instance variables private too.

Reply:
Ah fixed